pytroll_collectors/region_collector.py

```diff
@@ -102,15 +102,14 @@ def collect(self, granule_metadata):
                 self.granule_times.add(ptime)
                 self.granules.append(granule_metadata)
                 self.last_file_added = True
-                LOG.info("Added %s (%s) granule to area %s",
+                LOG.info("Added %s (%s) granule to area %s because we expect it",
```

Comment:
```suggestion
                LOG.info("Added overlapping granule %s (%s) to area %s",
```

Reply:
Chose a slightly different formulation instead.  One aim was that I wanted to understand from my logs whether granules were added because they were expected, or because they were newly overlapping.